### PR TITLE
the real 4.10.x changes

### PR DESCRIPTION
**Pull request by nhart**

### CODE REVIEW

**Comment by jordandukart** [comment]
Diffing, the `schema.xml` appears to be missing entries for `PID_kw` as well as `or_*`.

File: conf/solrconfig.xml [hunk 1]
```diff
@@ -783,7 +783,7 @@
 <str name="fl">*</str>
 <str name="q.alt">*:*</str>
 <str name="qf">dc.title^5 dc.subject^3 dc.description^3 dc.creator^3 dc.contributor^3 dc.type^1 dc.relation^1 dc.publisher^1 mods_identifier_local_ms^3 ds.WARC_FILTER^1 text_nodes_HOCR_hlt^1 mods_subject_hierarchicalGeographic_region_ms^3 mods_identifier_hdl_mt^3 dc.identifier^3 PID_kw^0.5 catch_all_fields_mt^0.1</str>
```
Comment (jordandukart) [inline conf/solrconfig.xml:785]: On the upstream `4.10.x` branch `PID_kw` is used as opposed to PID.

File: islandora_transforms/RELS-INT_to_solr.xslt [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!-- RELS-INT
+<!-- RELS-INT
```
Comment (jordandukart) [inline islandora_transforms/RELS-INT_to_solr.xslt]: Random whitespace?

File: islandora_transforms/RELS-EXT_to_solr.xslt [hunk 3]
```diff
@@ -4,7 +4,7 @@
     xmlns:java="http://xml.apache.org/xalan/java"
     xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
     xmlns:foxml="info:fedora/fedora-system:def/foxml#"
-    xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+    xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
```
Comment (jordandukart) [inline islandora_transforms/RELS-EXT_to_solr.xslt]: Random whitespace?

File: islandora_transforms/or_transcript_solr.xslt [hunk 4]
```diff
@@ -8,14 +8,14 @@
         <xsl:for-each select="$content//cue">
                 <field>
                     <xsl:attribute name="name">
-                        <xsl:value-of select="concat($prefix, 'cue_id')"/>
+                        <xsl:value-of select="concat($prefix, 'cue_id')"/>
```
Comment (jordandukart) [inline islandora_transforms/or_transcript_solr.xslt]: Moar random whitespace?

File: islandora_transforms/LICENSE.txt [hunk 5]
```diff
@@ -0,0 +1 @@
+{"error":"Not Found"}
```
Comment (jordandukart) [inline islandora_transforms/LICENSE.txt:1]: Wat's up with this?

**Review by jordandukart — CHANGES_REQUESTED** [review]

File: islandora_transforms/LICENSE.txt (hunk 5 above)
Comment (nhart) [reply islandora_transforms/LICENSE.txt:1]: looks like it's busted in core repo.. all the forks have this.

**Review by IAMlKeno — APPROVED** [review]

**Pull request merged**